config/flownode.example.toml

```diff
@@ -5,6 +5,9 @@ mode = "distributed"
 ## @toml2docs:none-default
 node_id = 14
 
+## The number of flow worker in flownode
+num_workers = 2
```

Comment:
What about supports `auto` value, just like many options in mito engine?

Reply:
> What about supporting `auto` value, just like many options in mito engine?

supporting both str("auto") and number will be unnecessarily complex, how about just say not set it will use auto default value in docs?

Reply:
You can modify the example to:

```toml
#+num_workers=2
```

This will be rendered correctly but treated as a comment.